tokio-trace/tokio-trace-core/src/lib.rs

```diff
@@ -98,7 +98,7 @@ macro_rules! identify_callsite {
 /// # #[macro_use]
 /// # extern crate tokio_trace_core;
 /// # use tokio_trace_core::{callsite::Callsite, subscriber::Interest};
-/// use tokio_trace_core::{Metadata, Level};
+/// use tokio_trace_core::metadata::{Metadata, Level, Kind};
```

Comment:
order here should be `use tokio_trace_core::metadata::{Kind, Level, Metadata};`

Reply:
Ah, yeah, I knew this was wrong but I had forgotten that rustfmt doesn't know about doctests.